exopy_hqc_legacy/instruments/drivers/visa_tools.py

```diff
@@ -153,13 +153,13 @@ def read_values(self, format=0):
         """
         return self._driver.read_values(format)
 
-    def ask(self, message):
+    def ask(self, message, *args, **kwargs):
```

Comment:
Why *args and **kwargs, query only has delay ?

Reply:
Future proofing I guess. I'd like this method to behave exactly like in the pyvisa docs. If you think we should control which arguments can be forwarded to pyvisa, I'll change it.